addons/im_livechat/tests/test_get_mail_channel.py

```diff
@@ -46,6 +51,9 @@ def test_get_mail_channel(self):
             channel_operator_ids = [channel_operator[0] for channel_operator in channel_operators]
             self.assertTrue(all(partner_id in channel_operator_ids for partner_id in self.operators.mapped('partner_id').ids))
 
+        visitor_user_channel = self.livechat_channel._get_mail_channel('Visitor', user_id=self.visitor_user.id)
+        self.assertEqual(visitor_user_channel['correspondent_name'], self.visitor_user.name, "Chat title should be correct")
```

Comment:
you didn't apply my comment about `visitor_user_channel['correspondent_name']`, did you?